paasta_tools/cli/utils.py

```diff
@@ -524,7 +528,11 @@ def execute_paasta_serviceinit_on_remote_master(subcommand, cluster, service, in
         return (
             'ERROR: could not find connectable master in cluster %s\nOutput: %s' % (cluster, output)
         )
-    return run_paasta_serviceinit(subcommand, master, service, instances, cluster, stream, **kwargs)
+    if ignore_ssh_output:
```

Comment:
ignore_ssh_output is kinda a misnomer right? It isn't really ignoring the ssh output as much as the... ssh logging?

Do you know why INFO was too noisy to create that "disconnection" message before? It didn't used to do that. Any particular reason not to just do QUIET all the time?